cclib/bridge/cclib2horton.py

```diff
@@ -80,7 +80,52 @@ def makehorton(data):
             attributes["npa_charges"] = data.atomcharges["natural"]
 
     elif hortonver == 3:
-        pass  # Populate with bridge for horton 3 in later PR
+        # In horton 3 IOData, inputs are type-checked -- thus the bridge also verifies the types.
+        # if coordinates are known; numpy.ndarray (size natom-by-3)
+        if hasattr(data, "atomcoords"):
+            attributes["atcoords"] = numpy.asanyarray(data.atomcoords)[-1]
+        # if atomic numbers are known; numpy.ndarray (size natom)
+        if hasattr(data, "atomnos"):
+            attributes["atnums"] = numpy.asanyarray(data.atomnos)
+        # if orbital coefficients known; iodata.orbitals.MolecularOrbitals
+        if hasattr(data, "mocoeffs"):
+            moattr = {
+                "kind": "restricted",
+                "norba": len(data.mocoeffs[0]),
+                "norbb": None,
+                "occs": numpy.concatenate(
+                    numpy.ones(data.homos[0]), numpy.zeros(len(data.mocoeffs[0]) - data.homos[0])
```

Comment:
Isn't `len(data.mocoeffs[0]) == data.nbasis`?

Reply:
Yes. Thank for pointing that out. I've just replaced it.